### DISCUSSION TITLE
Meshtastic integration

### DISCUSSION THREAD

**Discussion by Laarge** [andreynovikov/trekarta#266] [Ideas]
Hello 
I want to suggest a Meshtastic integration to be developed
https://meshtastic.org/ and https://github.com/meshtastic 
Your app will become extremely popular if you add such integration.
Imagine not.just you, but a whole group of hikers, visualized on the map.

**Comment by andreynovikov** [comment]
It's n interesting idea, but there is no way I will be able to get such device. And developing integration without ability to test it is a pointless idea.
